packages/core/scss/components/_checkboxes.scss

```diff
@@ -40,8 +42,8 @@
 
   [type='checkbox'] + label:before {
     content: '';
-    width: $size;
-    height: $size;
+    width: 18px;
```

Comment:
@goncalosard same as with radiobutton we check that the `$size` variable should be updated instead of the inline px value